library/core/src/lib.rs

```diff
@@ -164,6 +164,7 @@
 #![feature(const_waker)]
 #![feature(core_panic)]
 #![feature(duration_consts_float)]
+#![feature(generic_const_exprs)]
```

Comment:
`generic_const_exprs` is not something we're willing to enable for `core` at this time, as I understand it, so I'm going to close this.

(I would *also* like to have a function like this, but we need to figure out more about the `const` rules before that can happen.)